php/laravel-laravel-s/composer.json

```diff
@@ -0,0 +1,16 @@
+{
+  "require": {
+    "hhxsv5/laravel-s": "^3.7",
+    "laravel/framework": "^8.12"
+  },
+  "config": {
+    "optimize-autoloader": true,
```

Comment:
not need (this step is done on _containers_) 
https://github.com/the-benchmarker/web-frameworks/blob/e1350bceff5cc31fad0191ab63c55a670d309903/php/Dockerfile#L30

Reply:
OK